external/upstream/pybind11/CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ else()
         GIT_TAG v2.0.0
         UPDATE_COMMAND ""
         CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+                   -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+                   -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
```

Comment:
Yeah, I didn't want Joe to have a non-working codebase if the rebase didn't work. Good catch.